pkg/pillar/cmd/tpmmgr/tpmmgr.go

```diff
@@ -65,6 +68,33 @@ var (
 	}
 )
 
+var vendorRegistry = map[uint32]string{
```

Comment:
Oh and one more thing -- it would be useful to either use (or at least point out) the fact that these byte sequences are actually ASCII representations of vendor IDs.